<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="screen" cwd="node"/>
    <rosparam param="tl_graph_file">./light_classification/models/frozen_inference_graph.pb</rosparam>
    <rosparam param="tl_classes">
    [{"id":3, "name":"green"},{"id":1, "name":"red"},{"id":2, "name":"yellow"},{"id":4,"name":"off"}]
    </rosparam>
    <rosparam param="tl_threshold">0.9</rosparam>
</launch>